hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestS3GuardToolDynamoDB.java

```diff
@@ -324,4 +324,20 @@ public void testCLIFsckFailInitializeFs() throws Exception {
         () -> run(S3GuardTool.Fsck.NAME, "-check",
             "s3a://this-bucket-does-not-exist-" + UUID.randomUUID()));
   }
+
+  @Test
+  public void testCLIFsckDDbInternalWrongS3APath() throws Exception {
+    intercept(FileNotFoundException.class, "wrong path",
+        () -> run(S3GuardTool.Fsck.NAME, "-"+Fsck.DDB_MS_CONSISTENCY_FLAG,
+            "s3a://" + getFileSystem().getBucket() + "/" + UUID.randomUUID()));
+  }
+
+  @Test
+  public void testCLIFsckDDbInternalParam() throws Exception {
+    final int result = run(S3GuardTool.Fsck.NAME, "-"+Fsck.DDB_MS_CONSISTENCY_FLAG,
+        "s3a://" + getFileSystem().getBucket());
+    LOG.info("This test serves the purpose to run fsck with the correct " +
```

Comment:
move the first two lines of the log up into a describe() statement; if something fails it would be good to have it in the logs.